client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -383,4 +388,21 @@ class DomainItem extends PureComponent {
 	}
 }
 
-export default localize( DomainItem );
+const addEmailClick = ( domain ) =>
+	composeAnalytics(
+		recordGoogleEvent(
+			'Domain Management',
+			'Clicked "Add Email" Button in DomainItem',
+			'Domain Name',
+			domain.name
+		),
+		recordTracksEvent( 'calypso_domain_management_domain_item_add_email_click', {
+			section: domain.type,
+		} )
+	);
+
+export default connect( null, ( dispatch ) => {
+	return {
```

Comment:
I think it's preferred to declare the `mapDispatchToProps` as object when it just simply decorates the functions with dispatch